core-services/prow/02_config/openshift/route-controller-manager/_pluginconfig.yaml

```diff
@@ -2,8 +2,6 @@ label:
   restricted_labels:
     openshift/route-controller-manager:
     - allowed_users:
-      - akram
-      - sayan-biswas
       - ingvagabund
       - soltysh
       - stlaz
```

Comment:
Please also remove sttts and s-urbaniak, and add me (@Miciah) and @candita from the Network Edge team.

Reply:
Sure, we can file a followup for updating the route-controller-manager config.